components/802.15.4_RF/stm-s2lp-rf-driver/source/NanostackRfPhys2lp.cpp

```diff
@@ -695,7 +733,9 @@ static int8_t rf_extension(phy_extension_type_e extension_type, uint8_t *data_pt
             }
             break;
         case PHY_EXTENSION_SET_TX_POWER:
-            // TODO: Set TX output power
+            if (IS_PAPOWER_DBM(*data_ptr)) {
```

Comment:
This extension (PHY_EXTENSION_SET_TX_POWER) is actually not giving dBm value. It is used to give TX power as percentage of max output power. So you may get anything between 0 and 100 here.